apps/api/src/models/post.ts

```diff
@@ -0,0 +1,50 @@
+import mongoose, { Document, Schema } from 'mongoose';
+import PostComment from './postComment';
+
+interface IPost extends Document {
+    title: string,
+    image: string,
+    description: string,
+    category: object,
```

Comment:
Here instead of object we can use directly `mongoose.Types.ObjectId` same for comments `mongoose.Types.ObjectId[]`